src/neuvector/chart/templates/uds-package.yaml

```diff
@@ -4,6 +4,25 @@ metadata:
   name: neuvector
   namespace: {{ .Release.Namespace }}
 spec:
+  sso:
+    - name: Neuvector
+      clientId: uds-core-admin-neuvector
+      redirectUris:
+        - "https://neuvector.admin.{{ .Values.domain }}/openId_auth"
+      secretName: neuvector-secret
+      secretTemplate:
+        oidcinitcfg.yaml: |-
```

Comment:
Did you see any weird timing issues here/potential for them (i.e. similar to grafana failing to mount, but worse in this case since the volume is optional)? 

Another thing I'm thinking is that we no longer benefit from the upstream checksum on this secret to cycle the deployment. I'm not sure of a good answer, but it would be nice to ensure the deployment rolls if this secret changes.

Reply:
Added a patch to set a checksum annotation for the deployment.